### ISSUE TITLE
Annotation for the frame field

### ISSUE THREAD

**Issue by kriti115** [Lydorn/Polygonization-by-Frame-Field-Learning#30]
I have the angles for each pixel as a .npy file to be used as groundtruth as mentioned in the paper. 

My question is: is this directly used in the network or is there a need to calculate frame field to be fed into the network? If so what should the format be? 

I would appreciate any kind of help. Thank you.

**Comment by patriksabol** [comment]
The angles to be fed into the network are generated during training data generation process implemented as data_transforms (f.e. https://github.com/Lydorn/Polygonization-by-Frame-Field-Learning/blob/master/dataset_folds.py). You just need to adjust transformation to your own data.

**Comment by kriti115** [comment]
@patriksabol Thank you for you input. I was able to understand that we only need the images and binary masks as input and not the angles as they will be calculated while running the network.

I was able to run the network and got the training data as '.pt' file in processed folder. Although the training stopped with error half way I was still able to get half of the data in processed folder. 

I will close this issue now.
